### PR TITLE
Data section update

### PR DESCRIPTION
**Pull request by abuiquoc**

### CODE REVIEW

File: Data/collections.json [hunk 1]
```diff
@@ -5871,21 +5889,19 @@
                         "SpecificProduct":"Europe",
                         "Spatial": "Europe",
                         "Temporal":"2017",
```
Comment (Pratichhya) [inline Data/collections.json:5891]: The comma at the end of the Temporal key value is causing the action failure, I guess

File: Data/collections.json [hunk 2]
```diff
@@ -5871,21 +5889,19 @@
...
-                        "Catalogue":"[Link](https://catalogue.dataspace.copernicus.eu/resto/api/collections/S2GLC/search.json?){target='_blank'}"
                     },
                     {
                         "SpecificProduct":"Poland",
                         "Spatial": "Poland",
                         "Temporal":"2019 - 2021",
```
Comment (Pratichhya) [inline Data/collections.json:5896]: same as above

File: Data/collections.json [hunk 3]
```diff
@@ -3677,19 +3691,18 @@
...
                     {   
                         "ProductID":"CORINE LAND COVER (CLC)",
                         "Product_type":"CHANGES",
                         "SpecificProduct":"CHA_1990_2000, \n\n CHA_2000_2006, \n\n CHA_2006_2012, CHA_2012_2018",
-                        "Spatial": "Europe",
-                        "Temporal":"(*) Apr 1998 - Sep 2021",
+                        "Spatial": "Europe (*)",
                         "Catalogue":"/eodata/CLMS/Pan-European/CORINE_Land_Cover/",
                         "ProductLink":"[Details](https://land.copernicus.eu/pan-european/corine-land-cover){target='_blank'}"
+                        "Note": "(*) Number of involved countires (CLC_1990 – 27; CLC_2000 – 39, CLC_2006 – 39; CLC_2012 – 39; CLC 2018 – 39."
```
Comment (Pratichhya) [inline Data/collections.json:3705]: Issue with the symbol
```suggestion
                        "Note": "(*) Number of involved countires (CLC_1990 - 27; CLC_2000 - 39, CLC_2006 - 39; CLC_2012 - 39; CLC 2018 - 39."
                    
```

**Comment by abuiquoc** [comment]
@Pratichhya can we merge now?

**Review by Pratichhya — APPROVED** [review]

**Pull request merged**